doc/reference/lora_lorawan/index.rst

```diff
@@ -1,26 +1,53 @@
-.. _lorawan_api:
+.. _lora_lorawan_api:
 
-LoRaWAN
-#######
+LoRa and LoRaWAN
+################
 
 Overview
 ********
 
-Long Range Wide Area Network (LoRaWAN) is a network layer protocol based on
-the LoRa PHY.
+LoRa (abbrev. for Long Range) is a proprietary low-power wireless
```

Comment:
`LoRa (abbrev. for Long Range)` to `Long Range (LoRa)`

Reply:
I think it should be kept as is. The term "long range" could mean anything, but LoRa is a fixed brand, which stands for "long range". You would never say: I'm building a product with long range.

Reply:
I fixed the abbreviation to this context... But no issues though.